### PR TITLE
WIP: Add Bucket Wiping lambda to delete objects in bucket

### PR DESCRIPTION
**Pull request by stevededalus**
Iterate through every object in a bucket and delete it. Used for removing the config bucket on stack delete.

~~Open question: Should config bucket be unversioned or should this iterate over versioned objects?~~
Open question: Best way to write tests against lambdas?

### CODE REVIEW

File: lambdas/bucket-wiper/index.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```
Comment (dimaryaz) [inline lambdas/bucket-wiper/index.py]: Don't need this - that's not an executable script

File: lambdas/bucket-wiper/index.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+        if event['RequestType'] == 'Delete':
+            s3 = boto3.resource('s3')
+            bucket = s3.Bucket(bucket)
+            # for obj_version in bucket.object_versions.all():
+            versioned_objs = []
+            for obj_version in bucket.object_versions.all():
+                versioned_objs.append({'Key': obj_version.object_key,
+                                       'VersionId': obj_version.id})
+            bucket.delete_objects(Delete={'Objects': versioned_objs})
```
Comment (dimaryaz) [inline lambdas/bucket-wiper/index.py]: It supports up to 1000 objects at a time - so need to do it in chunks

Comment (stevededalus) [reply lambdas/bucket-wiper/index.py]: Hopefully we won't have 1000 objects in the config bucket, but updated just in case.

**Comment by stevededalus** [comment]
Successfully created and destroyed:
quilt-t4-staging-d5a4c0

**Pull request merged**